plugins/grpc/java/v1.65.1/buf.plugin.yaml

```diff
@@ -0,0 +1,25 @@
+version: v1
+name: buf.build/grpc/java
+plugin_version: v1.65.1
+source_url: https://github.com/grpc/grpc-java
+integration_guide_url: https://grpc.io/docs/languages/java/quickstart
+description: Generates Java client and server stubs for the gRPC framework.
+deps:
+  - plugin: buf.build/protocolbuffers/java:v25.3
```

Comment:
Any idea when we'll bump this?

Reply:
Waiting on https://github.com/grpc/grpc-java/issues/11015.